composer.json

```diff
@@ -6,11 +6,11 @@
     "require": {
         "cweagans/composer-patches": "^1.6.4",
         "drupal-composer/drupal-scaffold": "^2.0.0",
-        "drupal/lightning_api": "3.x-dev",
-        "drupal/lightning_core": "4.x-dev",
-        "drupal/lightning_layout": "1.x-dev",
-        "drupal/lightning_media": "3.x-dev",
-        "drupal/lightning_workflow": "3.x-dev",
+        "drupal/lightning_api": "^3.5",
+        "drupal/lightning_core": "^4.0-alpha1",
+        "drupal/lightning_layout": "^1.6",
+        "drupal/lightning_media": "^3.7",
```

Comment:
Same as workflow, I don't thin media 3.7 allows LC 4.x. We should change to `^3.8`.